index.html

```diff
@@ -0,0 +1,229 @@
+      </nav>
+    </header>
+    <main>
+      <section class="header-main">
+        <h1>I’m Solomon Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h4>LET'S CONNECT</h4>
+        <ul>
+          <li>
+            <a href="https://twitter.com/SolomonAfutu"
+              ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+              ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://github.com/SolomonTheCoder"
+              ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+        </ul>
+      </section>
+      <section class="Cards">
+        <div class="Card1">
+          <img src="img/Snapshoot Portfolio.png" alt="" />
+          <div class="Card1-P">
+            <h4>Tonic</h4>
+            <div class="Card1-tech">
+              <div>
+                <p class="P1">Canopy</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P2">Back End Dev</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P3">2015</p>
+              </div>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
+        <div class="Card2">
+          <img src="img/Snapshoot Portfolio (1).png" alt="" />
+          <div class="Card2-P">
+            <h4>Multi-Post Stories</h4>
+            <div class="Card2-tech">
+              <div>
+                <p class="P1">Canopy</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P2">Back End Dev</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P3">2015</p>
+              </div>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
```

Comment:
-  [ ] Kindly follow the Figma Template (Ex: color, font size, etc...)

- Your Design
<img width="497" alt="Screenshot 2023-03-23 at 16 00 00" src="https://user-images.githubusercontent.com/81444338/227384987-c16d3b78-6300-487e-9c7c-7866eb820f7a.png">

- [ ] Figma  Design
<img width="1440" alt="Screenshot 2023-03-23 at 16 02 57" src="https://user-images.githubusercontent.com/81444338/227385047-67581978-ebd7-444c-89c5-c46ae3f23ce2.png">